includes/class-bitcoin-donation-shortcode-wide.php

```diff
@@ -12,25 +12,40 @@ public function __construct()
 
     function bitcoin_donation_render_shortcode_wide()
     {
-        $options = get_option('bitcoin_donation_options');
+        $options = get_option('bitcoin_donation_forms_options');
         $options = is_array($options) ? $options : [];
-        $theme_class = isset($options['theme']) && $options['theme'] === 'dark' ? 'bitcoin-donation-dark-theme-wide' : 'bitcoin-donation-light-theme-wide';
+        $theme_class = isset($options['theme']) && $options['theme'] === 'dark' ? 'bitcoin-donation-dark-theme' : 'bitcoin-donation-light-theme';
         $currency = $options['currency'] ?? 'USD';
         $butoon_text = $options['button_text'] ?? 'Donate';
         $title_text = $options['title_text'] ?? 'Donate with Bitcoin';
 
         ob_start();
 ?>
-        <div id="bitcoin-donation-donation-form-wide" class="<?php echo esc_attr($theme_class); ?>">
-            <div class="bitcoin-donation-title-wrapper-wide">
+        <div id="bitcoin-donation-form-wide" class="bitcoin-donation-donation-form <?php echo esc_attr($theme_class); ?> wide-form">
+            <div class="bitcoin-donation-title-wrapper">
                 <h3><?php echo esc_html($title_text); ?></h3>
 
             </div>
+            <input type="text" id="bitcoin-donation-email-wide" name="bitcoin-email" style="display: none;" aria-hidden="true">
             <div class="bitcoin-donation-wide-field-wrapper">
-                <input class="amount-field" type="number" id="bitcoin-donation-amount" step="0.01" placeholder="Amount (in <?php echo esc_html($currency); ?>)">
-                <input class="amount-field" type="number" id="bitcoin-donation-satoshi" placeholder="Satoshi">
-                <input class="bitcoin-donation-message-wide" id="bitcoin-donation-message" placeholder="Message"></input>
-                <button id="bitcoin-donation-pay"><?php echo esc_html($butoon_text); ?></button>
+                <div class="bitcoin-donation-wide-up">
+
+                    <div class="shoutout-input-label">
+                        <label for="bitcoin-donation-amount">Amount (in <?php echo esc_html($currency); ?>)</label>
```

Comment:
Label has wrong element reference.
```suggestion
                        <label for="bitcoin-donation-amount-wide">Amount (in <?php echo esc_html($currency); ?>)</label>
```